tests/config_tests_prov_bad.toml

```diff
@@ -1,35 +0,0 @@
-[gateway]
```

Comment:
It doesn't appear to be used anywhere. Am I wrong? If we are using it, I of course have no problem keeping it!